smart_card_connector_app/src/chrome-api-jstocxxtest.js

```diff
@@ -93,6 +94,19 @@ goog.exportSymbol('testChromeApiProviderToCpp', {
   'testSmoke': async function() {
     launchPcscServer(/*initialDevices=*/[]);
     await pcscReadinessTracker.promise;
+  },
+
+  'testEstablishContext': async function() {
+    launchPcscServer(/*initialDevices=*/[]);
+    await pcscReadinessTracker.promise;
```

Comment:
Is this necessary? Ideally we test without this, to verify that even early API events are handled correctly (by being delayed until the PC/SC server initialization completes).

Reply:
good catch!